proxy/envoy/testdata/cds-circuit-breaker.json.golden

```diff
@@ -71,35 +71,21 @@
     "service_name": "world.default.svc.cluster.local|custom",
     "connect_timeout_ms": 1000,
     "type": "sds",
-    "lb_type": "round_robin",
-    "max_requests_per_connection": 100,
-    "circuit_breakers": {
-     "default": {
-      "max_connections": 100,
-      "max_pending_requests": 100,
-      "max_requests": 100
-     }
-    },
-    "outlier_detection": {
-     "consecutive_5xx": 10,
-     "interval_ms": 30000,
-     "base_ejection_time_ms": 15500,
-     "max_ejection_percent": 100
-    }
+    "lb_type": "round_robin"
```

Comment:
? Why is circuit breaker removed ? According to the spec above, it should be present right?

Reply:
It's testing that CB only applies to version=v1 like it should.